File: mw_models/hernquist_test/make_init.py
import lilguys
import numpy as np

def main():
    p0 = 1
    v0 = 0.5
    r = np.array([2, 0, 0])

    v1 = np.array([0, v0, 0])
    v2 = 0.8*v1
    v3 = 1.2*v1

    pos = np.array([r, -r, r])
    vel = np.array([v1, v2, v3])
    m = 0

    s = lilguys.Snapshot(pos, vel, m=m)
    s.save("hernquist.hdf5")


if __name__ == "__main__":
    main()

File: models/tests/hernquist/random/make_init.jl
import LilGuys as lguys


function main()
    N = 1000

    r0 = 2
    v0 = 0.5
    rs = r0 .+ r0/5 .* randn(N)
    vs = v0 .+ v0/5 .* randn(N)

    pos = reshape(rs, 1, N) .* lguys.rand_unit(N)
    vel = reshape(vs, 1, N) .* lguys.rand_unit(N)

    snap = lguys.Snapshot(positions=pos, velocities=vel, masses=zeros(N))
    lguys.save("initial.hdf5", snap)
end


if abspath(PROGRAM_FILE) == @__FILE__
    main()
end
